verticapy/utilities.py

```diff
@@ -257,50 +298,67 @@ def drop(
             warnings.warn(warning_message, Warning)
             result = False
     elif method == "temp":
-        sql = "SELECT table_schema, table_name FROM columns WHERE LOWER(table_name) LIKE '%verticapy%' GROUP BY 1, 2;"
-        cursor.execute(sql)
-        all_tables = cursor.fetchall()
+        sql = "SELECT table_schema, table_name FROM columns WHERE LOWER(table_name) LIKE '%_verticapy_tmp_%' GROUP BY 1, 2;"
+        all_tables = result = executeSQL(sql, print_time_sql=False, method="fetchall")
         for elem in all_tables:
-            table = '"{}"."{}"'.format(
-                elem[0].replace('"', '""'), elem[1].replace('"', '""')
-            )
-            with warnings.catch_warnings(record=True) as w:
-                drop(
-                    table, cursor, method="table",
-                )
-        sql = "SELECT table_schema, table_name FROM view_columns WHERE LOWER(table_name) LIKE '%verticapy%' GROUP BY 1, 2;"
-        cursor.execute(sql)
-        all_views = cursor.fetchall()
+            table = '"{}"."{}"'.format(elem[0].replace('"', '""'), elem[1].replace('"', '""'))
+            drop_if_exists(table, method="table")
+        sql = "SELECT table_schema, table_name FROM view_columns WHERE LOWER(table_name) LIKE '%_verticapy_tmp_%' GROUP BY 1, 2;"
+        all_views = executeSQL(sql, print_time_sql=False, method="fetchall")
         for elem in all_views:
-            view = '"{}"."{}"'.format(
-                elem[0].replace('"', '""'), elem[1].replace('"', '""')
-            )
-            with warnings.catch_warnings(record=True) as w:
-                drop(
-                    view, cursor, method="view",
-                )
+            view = '"{}"."{}"'.format(elem[0].replace('"', '""'), elem[1].replace('"', '""'))
+            drop_if_exists(view, method="view")
         result = True
     else:
         result = True
     return result
 
+# ---#
+def drop_if_exists(name: str = "", method: str = "auto"):
+    """
+---------------------------------------------------------------------------
+Drops the input relation if it exists. This can be a model, view, table, 
```

Comment:
"This" should have the noun it's referring to after it. "This relation" ?

Reply:
It will be corrected in the next commit.